src/index.tsx

```diff
@@ -552,6 +558,10 @@ class RichMarkdownEditor extends React.PureComponent<Props, State> {
     return this.props.theme || (this.props.dark ? darkTheme : lightTheme);
   };
 
+  dictionary = () => {
+    return { ...baseDictionary, ...this.props.dictionary };
+  };
```

Comment:
This is my only concern with the PR, as it'll create a new object every render loop it'll potentially cause sub-components to have their props have changed.

Need to have some sort of memoization here, could use lodash or getDerivedStateFromProps?

Reply:
True, I'll take care of that. Thanks!